src/index.js

```diff
@@ -84,7 +84,7 @@ const addTrip = (tripDetails) => {
     })
     .then(response => response.json())
     .then(json => console.log(json))
-    .catch(err => console.log("Yikes"))
+    .catch(err => domUpdates.displayMessage("Sorry.  Your trip request didn't go through"))
```

Comment:
Good job on updating the DOM with a message if something goes wrong! If you have time maybe different messages for different errors would be nice